i2c_handler.py

```diff
@@ -1,18 +1,25 @@
+import sys
+import os
 import json
 import logging
 from threading import Lock, Thread
-from time import sleep
+import time
 from datetime import datetime, timezone
 
 # Hardware Libraries
 import board
 import busio
 import adafruit_ads1x15.ads1115 as ADS
 from adafruit_ads1x15.analog_in import AnalogIn
-from adafruit_lsm6ds.lsm6ds33 import LSM6DS33
-from adafruit_lsm6ds import Rate, AccelRange, GyroRange
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from imu_driver.adafruit_lsm6ds.lsm6ds33 import LSM6DS33
+from imu_driver.adafruit_lsm6ds import Rate, AccelRange, GyroRange
```

Comment:
I think its good to keep it in its own directory, makes the relation between the adafruit library and the imu controller clear, in the future it might be good to have a sensor directory with all the drivers for each device